css-typed-om/Overview.bs

```diff
@@ -2903,6 +2903,44 @@ depending on the property they came from:
         return "currentcolor".
     2. Otherwise, return the result of serializing the <<color>> value.
 
+: 'display'
+::
+    1. Let |values| initially be the empty [=list=].
+    2. If 'display-outside' or 'display-inside' is specified:
+        1. Append value of 'display-outside' to |values|.
+        2. Append value of 'display-inside' to |values|.
+        3. Goto step 7.
```

Comment:
This means you won't hit the list-item step at all if you specified something like `display: block list-item`.